geode-for-redis/src/integrationTest/java/org/apache/geode/redis/internal/commands/executor/pubsub/AbstractSubscriptionsIntegrationTest.java

```diff
@@ -74,9 +74,6 @@ public void pingWithArgumentWhileSubscribed() {
     executor.submit(() -> client.subscribe(mockSubscriber, "same"));
     mockSubscriber.awaitSubscribe("same");
     mockSubscriber.ping("potato");
-    // JedisPubSub PING with message is not currently possible, will submit a PR
-    // (https://github.com/xetorthio/jedis/issues/2049)
-    // until then, we have to call this second ping to flush the client
     mockSubscriber.ping();
```

Comment:
This second ping should also be removed, as it's no longer necessary, and the assertion on line 79 should be changed from `isEqualTo(2)` to `isOne()`